### ISSUE TITLE
[Accessibility]A11y_Java Tooling for IntelliJ/Eclipse_Source tab_Screen Reader: On changing the tab selection, Narrator is not reading the selected tab details

### ISSUE THREAD

**Issue by soumyatippapuram** [microsoft/azure-tools-for-java#4300]
[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier

GitHubTags: #A11yTCS; #A11yMAS; #DesktopApp; #Win10; #A11y_JavaToolingforIntelliJ/Eclipse-Benchmark; #MAS1.3.1; #ScreenReader; #JavatoolingforIntelliJ/Eclipse; #C+E-H; #Benchmark; #A11ySev1; #Narrator 

**NOTE:** same issue is observed when we are using tab key to navigate to selected tab, Narrator is still reading "Source" tab 

**Environment Details:**
Eclipse IDE for Enterprise Java Developers.
Version: 2020-03 (4.15.0)
Build id: 20200313-1211

**Solution Installed:** The Azure Toolkit for Eclipse 3.220, Scala IDE 4.7.x

**OS Details:**
OS: Windows (Version 1903, Build: 18362.175)

**Screen Reader:** Narrator

**MAS Violated:** MAS 1.3.1

**Repro Steps:**
1.Start Screen Reader. Open Java EE - Eclipse application 
2.Install "Azure Toolkit for Eclipse" and "Scala Plug-ins" from Eclipse Marketplace
3.Tab till "File > New > Project" in the navigation bar and press enter
4.Tab till "HDInsight Project > Spark on HDInsight (Scala)" and select it.
5.Tab till "Next" button and press enter 
6.Enter all the required details, tab till "Next" button and press enter
7.Tab till "Source" tab. Now using arrow keys, change the tab selection
8.Check if the Narrator is reading the selected tab details properly

**Actual Result:**
On changing the tab selection, Narrator is still reading about "Source" tab instead of reading the selected tab details

**NOTE:** NVDA is reading it properly

**Expected Result:**
When the selection is changed, Narrator should read about the newly selected tab details

**User impact:**
Screen Reader users will not understand whether they are in "Source" tab even after performing some action

**MAS Reference:**
[MAS 1.3.1](https://microsoft.sharepoint.com/:w:/r/sites/accessibility/_layouts/15/Doc.aspx?sourcedoc=%7B97358E88-4367-4121-BF9D-F0C6597D03B7%7D&file=MAS%201.3.1%20%E2%80%93%20Info%20and%20Relationships%20.docx&action=default&mobileredirect=true)

[Bug_4300_Java Tooling_Source_Narrator is reading selected tab as source.pptx](https://github.com/microsoft/azure-tools-for-java/files/4469354/Bug_4300_Java.Tooling_Source_Narrator.is.reading.selected.tab.as.source.pptx)

[Bug_4300_Java Tooling_Source Tab_Narrator is reading all tabs as Source.pptx](https://github.com/microsoft/azure-tools-for-java/files/4469365/Bug_4300_Java.Tooling_Source.Tab_Narrator.is.reading.all.tabs.as.Source.pptx)

![Bug_4300_Java Tooling_Source_NVDA_No Issue](https://user-images.githubusercontent.com/41108985/79118262-09789b00-7dab-11ea-9c2e-c6fadb968166.png)

**Comment by jingyanjingyan** [comment]
+ Importance of experience: Minimal. It does not block the create project process, and user could access the details under each tab.
+ Quality of experience: Annoying. Screen Reader users will not understand whether they are in "Source" tab even after performing some action
+ Alternate solutions: This issue will not break users experience, no need for alternate solutions
+ Evidence: All evidence is provided in this issue.
+ [AAT Link](https://cela-ra-aat-prod.azurewebsites.net/#/workflow/56452990-0d38-45bc-b475-a62832ce0076)

**Comment by soumyatippapuram** [comment]
I will get this reviewed by SME and take appropriate action on this.

**Comment by jingyanjingyan** [comment]
This wizard is extends from original Eclipse wizard. You can find the same dialog in New Project > Java > Java Project. So it's out of the scope of our toolkit. Please find the screenshot of Eclipse without our toolkit.
![image](https://user-images.githubusercontent.com/36756024/81391577-ea71ec80-914f-11ea-8467-a5fd27a7313a.png)

**Comment by Alexandria-R** [comment]
Known eclipse issue, approved to close

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749650
